### PR TITLE
Add a subsection in Getting Started doc to describe how to target a Kubernetes Service as a target_url

### PR DESCRIPTION
**Pull request by buraksekili**
## Description
This PR adds a new subsection under `Create API`, to describe how to target a Kubernetes service

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). If PRing from your fork, don't come from your `master`!
- [x] Make sure you are making a pull request against our **`master` branch** (left side). Also, it would be best if you started *your change* off *our latest `master`*.
- [ ] Make sure you are updating [CHANGELOG.md](../CHANGELOG.md) based on your changes.
- [x] My change requires a change to the documentation.
  - [ ] If you've changed APIs, describe what needs to be updated in the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] Check your code additions will not fail linting checks:
  - [x] `gofmt -s -w .`
  - [x] `go vet ./...`
  - [x] `golangci-lint run`

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -178,6 +178,120 @@ $ curl -i localhost:8080/httpbin/get
...
+Tyk Operator allows accessing your Kubernetes service as an upstream proxy target.
+Thus, you can set the `proxy.target_url` as a Kubernetes Service following [DNS for Services and Pods guideline](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/), 
+so that the requests will be proxied to your service.
+
+> In general, Kubernetes Services have `<service-name>.<namespace-name>.svc.cluster.local` DNS entry once they are created. 
+For example, if you have a service called `httpbin` in `default` namespace, you can contact `httpbin` service with 
+`httpbin.default.svc` DNS record in the cluster, instead of IP addresses. 
+Please visit the official [Kubernetes documentation](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/) for more details.
+
+Suppose we want to create a Deployment of popular [`httpbin` service](https://hub.docker.com/r/kennethreitz/httpbin/)
```
Comment (andrei-tyk) [inline docs/getting-started.md:192]: I think correct is "the popular" and without syntax highlighting on the "service" word

File: docs/getting-started.md [hunk 2]
```diff
@@ -178,6 +178,120 @@ $ curl -i localhost:8080/httpbin/get
...
+> In general, Kubernetes Services have `<service-name>.<namespace-name>.svc.cluster.local` DNS entry once they are created. 
+For example, if you have a service called `httpbin` in `default` namespace, you can contact `httpbin` service with 
+`httpbin.default.svc` DNS record in the cluster, instead of IP addresses. 
+Please visit the official [Kubernetes documentation](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/) for more details.
+
+Suppose we want to create a Deployment of popular [`httpbin` service](https://hub.docker.com/r/kennethreitz/httpbin/) 
+using [`ci/upstreams/httpbin.yaml`](../ci/upstreams/httpbin.yaml) file. We are going to expose the application through port `8000` as described under
+the Service [specification](https://github.com/TykTechnologies/tyk-operator/blob/master/ci/upstreams/httpbin.yaml#L10).
+
+First, let's create Service and Deployment either applying the manifest defined in our repository
```
Comment (andrei-tyk) [inline docs/getting-started.md:196]: "let's create the service and delpoyment by either applying ..."

File: docs/getting-started.md [hunk 3]
```diff
@@ -178,6 +178,120 @@ $ curl -i localhost:8080/httpbin/get
...
+          imagePullPolicy: IfNotPresent
+          name: httpbin
+          ports:
+            - containerPort: 80
+EOF
+```
+
+> Please wait awhile until all pods reach READY `1/1` and STATUS `Running` state.
+
+Once the pod is ready, we can update our `httpbin` API's `target_url` field for proxying our requests to the Service that we've created above.
```
Comment (andrei-tyk) [inline docs/getting-started.md:247]: "to proxi" sounds a bit better than "for proxying"

File: docs/getting-started.md [hunk 4]
```diff
@@ -178,6 +178,120 @@ $ curl -i localhost:8080/httpbin/get
...
+  protocol: http
+  active: true
+  proxy:
+    target_url: http://httpbin.default.svc:8000
+    listen_path: /httpbin
+    strip_listen_path: true
+EOF
+```
+
+Please pay attention to the value of `spec.proxy.target_url` field.
```
Comment (andrei-tyk) [inline docs/getting-started.md:274]: "to the value of the .... field"

Comment (buraksekili) [reply docs/getting-started.md]: Updated @andrei-tyk, thanks 🙏

**Review by andrei-tyk — APPROVED** [review]

File: docs/getting-started.md [hunk 5]
```diff
@@ -178,6 +178,120 @@ $ curl -i localhost:8080/httpbin/get
...
+    target_url: http://httpbin.default.svc:8000
+    listen_path: /httpbin
+    strip_listen_path: true
+EOF
+```
+
+Please pay attention to the value of the `spec.proxy.target_url` field. 
+It is set to `http://httpbin.default.svc:8000` by following the convention described above (`<service_name>.<namespace>.svc:<service_port>`).
+
+Now, if you send your request to the `/httpbin` endpoint, the request will be proxied to the `httpbin Service`.
```
Comment (asoorm) [inline docs/getting-started.md:277]: This could be a little confusing. It may be clearer to specify the `/httpbin` endpoint of the Tyk Gateway.

Comment (buraksekili) [reply docs/getting-started.md]: updated @asoorm, thanks

**Review by asoorm — APPROVED** [review]

**Pull request merged**